pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -233,9 +243,10 @@ type ReconcileClusterDeployment struct {
 // +kubebuilder:rbac:groups=core,resources=pods;namespaces,verbs=get;list;watch
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles;rolebindings,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterdeployments;clusterdeployments/status;clusterdeployments/finalizers,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterprovisions,verbs=get;list;watch;create
 // +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterimagesets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterimagesets/status,verbs=get;update;patch
-func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (result reconcile.Result, returnErr error) {
```

Comment:
Yes. I am including `result` in the "reconcile complete" info log so that we can see whether the requeueAfter field is what we expect.

https://github.com/openshift/hive/blob/f212ef54ec1a17c06fba131ee74d283edf6302d2/pkg/controller/clusterdeployment/clusterdeployment_controller.go#L259-L263

Reply:
Ack